### ISSUE TITLE
Crash in matches-prop-shorthand when importing lodash methods directly

### ISSUE THREAD

**Issue by Wilfred** [wix/eslint-plugin-lodash#107]
Given the JS:

```js
import {map} from 'lodash';

function foo() {
  map(bar, function (x) { return x; });
}
```

and the following .eslintrc.json:

```json
{
    "parserOptions": {
        "ecmaVersion": 6,
        "sourceType": "module"
    },
    "rules": {
        "lodash/matches-prop-shorthand": "warn"
    },
    "plugins": [
        "lodash"
    ]
}
```

I get an exception:

```
Cannot read property 'name' of undefined
TypeError: Cannot read property 'name' of undefined
    at getImportedLodashMethod (/users/is/whughes/workspace/grail/node_modules/eslint-plugin-lodash/lib/util/lodashUtil.js:201:47)
    at isCallToLodashMethod (/users/is/whughes/workspace/grail/node_modules/eslint-plugin-lodash/lib/util/lodashUtil.js:261:126)
    at Object.canUseShorthand (/users/is/whughes/workspace/grail/node_modules/eslint-plugin-lodash/lib/rules/matches-prop-shorthand.js:50:74)
    at always (/users/is/whughes/workspace/grail/node_modules/eslint-plugin-lodash/lib/util/lodashUtil.js:277:68)
    at EventEmitter.<anonymous> (/users/is/whughes/workspace/grail/node_modules/eslint-plugin-lodash/lib/util/lodashUtil.js:236:17)
    at emitOne (events.js:96:13)
    at EventEmitter.emit (events.js:188:7)
    at NodeEventGenerator.enterNode (/users/is/whughes/workspace/grail/node_modules/eslint/lib/util/node-event-generator.js:40:22)
    at CodePathAnalyzer.enterNode (/users/is/whughes/workspace/grail/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:608:23)
    at CommentEventGenerator.enterNode (/users/is/whughes/workspace/grail/node_modules/eslint/lib/util/comment-event-generator.js:97:23)
```

This did not occur on v1.10.1.

**Comment by Lalem001** [comment]
I am seeing this same issue with several other rules:

- prefer-is-nil
- matches-prop-shorthand
- matches-shorthand
- prop-shorthand

The smallest sample code I could come up with to reproduce the error is:

```javascript
'use strict';

const map = require('lodash/map');
const noop = require('lodash/noop');

const cbs = [noop, noop];

// the next line causes errors in matches-prop-shorthand && matches-shorthand
const result = map(cbs, (callback) => callback());
```

I am still working on samples for `prop-shorthand` and `prefer-is-nil`

**Comment by Lalem001** [comment]
Smallest sample to reproduce `prefer-is-nil` error:

```javascript
'use strict';

const anyLodashImport = require('lodash/any');

// The next line causes error in prefer-is-nil
module.exports = 'foo' || 'bar';
```

**Comment by ganimomer** [comment]
Fixed in version `2.1.1`.
